pci/graphic_drivers/profiles.toml

```diff
@@ -27,7 +27,7 @@ conditional_packages = """
     if [ -n "$kernels" ]; then
         for kernel in $kernels; do
             case "$kernel" in
-                *-headers|*-nvidia|*-zfs);;
+                *-headers|*-zfs);;
```

Comment:
this results in `linux-cachyos-nvidia linux-cachyos-nvidia-nvidia`
if nvidia module already installed

Reply:
yep, my bad. We still need to add packages to the list that are installed when we remove them. https://github.com/CachyOS/chwd-db/pull/26/commits/57c78cb241f6538e3602296cabfd0b7eaec4c445 should fix that.